front/playwright.config.ts

```diff
@@ -12,7 +12,7 @@ export default defineConfig({
   testDir: './tests',
 
   /* Maximum time one test can run for. */
-  timeout: 50 * 1000,
+  timeout: 60 * 1000,
```

Comment:
Why do you need to increase the timeout timer ? Same for the retries

Reply:
We keep getting warnings that some old tests are slow in the CI.
![image](https://github.com/OpenRailAssociation/osrd/assets/17015973/83d9274a-b024-4a0e-9a71-056c54d65b55)
 To avoid any risk of timeout problems and upcoming tests for search, I increased the timeout to 1 minute. For retries, I set it to 3 to anticipate potential failures related to changes in the code for V1/V2